elixir/apps/domain/lib/domain/tokens/token.ex

```diff
@@ -1,11 +1,16 @@
-# TODO: service accounts auth as clients and as API clients?
+# TODO:  xxx service accounts auth as clients and as API clients?
```

Comment:
I suspect customers will want to keep these segregated to keep the two security contexts segregated, but we can discuss further when we start on #2027

Reply:
That is exactly why I left this comment: I think we will just add a new actor type.